warehouse/manage/views.py

```diff
@@ -1073,24 +1222,24 @@ def manage_project_history(project, request):
     except ValueError:
         raise HTTPBadRequest("'page' must be an integer.")
 
-    journals_query = (
-        request.db.query(JournalEntry)
-        .options(joinedload("submitted_by"))
-        .filter(JournalEntry.name == project.name)
-        .order_by(JournalEntry.submitted_date.desc(), JournalEntry.id.desc())
+    events_query = (
```

Comment:
I think we still want to display the JournalEntry data in this view, until we decide on how we'll handle either migrating this data into ProjectEvent objects, or otherwise deprecating it.

Reply:
Hmm, displaying both the `JournalEntry` and `ProjectEvent` tables in the same view might be a bit difficult because of the pagination generator. Would a separate tab like "Legacy Journal" (or similar) be alright?

Reply:
A separate tab is fine with me.

Reply:
Certainly not in the same table, no. A separate Tab seems best and cleanest until we decide the fate of Journal

Reply:
Done!